_scss/_content.scss

```diff
@@ -129,10 +129,9 @@ pre {
     font-size: 13px;
     line-height: 1.42857143;
     color: #333;
-    word-break: normal!important; 
-    word-wrap: break-word; 
+    word-break: normal!important;
```

Comment:
You should configure atom not to change white space in lines you didn't otherwise touch.